.github/workflows/create-release-pr.yaml

```diff
@@ -0,0 +1,328 @@
+              failed_plugins+=("$plugin")
+            fi
+          done
+
+          for plugin in "${unit_test_plugins[@]}"; do
+            if [[ ! " ${failed_plugins[@]} " =~ " $plugin " ]]; then
+              failed_plugins+=("$plugin")
+            fi
+          done
+
+          for plugin in "${integration_test_plugins[@]}"; do
+            if [[ ! " ${failed_plugins[@]} " =~ " $plugin " ]]; then
+              failed_plugins+=("$plugin")
+            fi
+          done
+
+          echo "Failed plugins: ${failed_plugins[*]}"
+
+          # Get release for the branch name and docs
+
+          release_version="${{steps.current_available_version.outputs.current_available_version}}"
+          echo "Remote version = $release_version"
+
+          # Configure the git bot
+
+          git config --global user.name 'Release Bot'
+          git config --global user.email 'release-bot@users.noreply.github.com'
+          git remote set-url --push origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/hyperledger/aries-acapy-plugins
+          git fetch --all
+          git checkout -b "release-v$release_version"
+
+          # Add all the changed files and then remove the failed plugins
+
+          git add .
+
+          for plugin in "${failed_plugins[@]}"; do
+            git restore --staged $plugin
+            git checkout -- $plugin
+          done
+
+          # Get the release notes and update the plugin decription with acapy version
+
+          body=$(python repo_manager.py 4)
+
+          # Update the RELEASES.md file with the release notes
+
+          echo "$body" | cat - RELEASES.md > temp && mv temp RELEASES.md
+
+          # Add the updated pyproject.toml and RELEASES.md files
+          git add ./*pyproject.toml
+          git add ./RELEASES.md
+
+          # Check if there are any upgrades
+          upgraded_plugins=$(python repo_manager.py 5)
+          has_upgrades=false
+
+          echo 
+          for plugin in ${potential_upgrades[@]}; do
+            for upgrade in ${upgraded_plugins[@]}; do
+              if [[ $plugin == *"$upgrade"* ]]; then
+                has_upgrades=true
+                break
+              fi
+            done
+          done
+
+          if [ "$has_upgrades" = true ]
+          then
+            echo "Upgrades detected. Committing the changes"
+          else
+            echo "No upgrades detected. Skipping commit and release"
+            exit 1
+          fi
+
+          git commit -s -m "Release v$release_version Upgrades"
+          git push --set-upstream origin "release-v$release_version"
+
+          # Create the PR
+
+          details="Plugins upgraded this release: "
+          for plugin in ${upgraded_plugins}; do
+            details="$details $plugin"
+          done
+
+          title="Release for aries-cloudagent v$release_version"
+          gh pr create --title "$title" --body "$body $details" --base main --head "release-v$release_version" 
```

Comment:
Few things here:
- Would it be better to use conditional logic on the step rather than checking `steps.integration_test_plugins.outputs.integration_test_exit_code` inside the script?
- `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` environemnt variable is not used.  It's better to use `${{ secrets.GITHUB_TOKEN }}` directly anyway.
- There's a better way to update code and submit a PR.  See https://github.com/hyperledger/indy-node/blob/main/.github/workflows/tag.yaml